java/src/test/java/org/rocksdb/RocksDBTest.java

```diff
@@ -602,6 +602,60 @@ public void compactRangeToLevel()
     }
   }
 
+  @Test
+  public void deleteFilesInRange()
+      throws RocksDBException, InterruptedException {
+    final int KEY_SIZE = 20;
+    final int VALUE_SIZE = 3000;
+    final int FILE_SIZE = 100000;
+    final int NUM_FILES = 10;
+    
+    final int KEY_INTERVAL = 10000;
+    try (final Options opt = new Options().
+        setCreateIfMissing(true).
+        setCompactionStyle(CompactionStyle.LEVEL).
+        setCompressionType(CompressionType.NO_COMPRESSION).
+        setTargetFileSizeBase(FILE_SIZE).
+        setTargetFileSizeMultiplier(1).
+        setWriteBufferSize(FILE_SIZE / 2).
+        setNumLevels(5).
+        setDisableAutoCompactions(true);
+         final RocksDB db = RocksDB.open(opt,
```

Comment:
I think several of these options and unnecessary for the test and could be removed, to make the intention clearer.